tests/data/virtual-server-upstream-options/plus-virtual-server-with-invalid-keys-openapi.yaml

```diff
@@ -19,7 +19,8 @@ spec:
       enable: "yes"
       path: 200
       interval: 1.5
-      jitter: 2.0
+      jitter: 2.
```

Comment:
Zero looks to have been removed here by mistake.

Reply:
Good catch!